Maya/Exporter/BabylonExporter.Mesh.cs

```diff
@@ -509,14 +509,8 @@ private BabylonNode ExportMesh(MDagPath mDagPath, BabylonScene babylonScene)
             //var hasColor = unskinnedMesh.NumberOfColorVerts > 0;
             //var hasAlpha = unskinnedMesh.GetNumberOfMapVerts(-2) > 0;
 
-            // TODO - Add custom properties
-            //var optimizeVertices = false; // meshNode.MaxNode.GetBoolProperty("babylonjsexportParameters.optimizeVertices");
+            //var optimizeVertices = meshNode.MaxNode.GetBoolProperty("babylonjsexportParameters.optimizeVertices");
             var optimizeVertices = exportParameters.optimizeVertices; // global option
-            if (optimizeVertices && hasMorphTarget)
```

Comment:
Was this fixed via your fix of the morph target rebuilding logic via https://github.com/BabylonJS/Exporters/pull/908?

Reply:
Suppose to be, the guy's use it sucessfully with a custom version removing these lines. 
However, following one of our previous meeting, we have to keep in mind that optimize vertices option HAS TO be removed because it's the normal way to do it.